demos/background_subtraction_demo/cpp_gapi/main.cpp

```diff
@@ -147,13 +148,13 @@ int main(int argc, char* argv[]) {
         auto config = ConfigFactory::getUserConfig(FLAGS_d, FLAGS_nireq, FLAGS_nstreams, FLAGS_nthreads);
         // clang-format off
         const auto net =
-            cv::gapi::ie::Params<cv::gapi::Generic>{
+            cv::gapi::ov::Params<cv::gapi::Generic>{
                 model->getName(),
                 FLAGS_m,  // path to topology IR
                 fileNameNoExt(FLAGS_m) + ".bin",  // path to weights
                 FLAGS_d  // device specifier
             }.cfgNumRequests(config.maxAsyncRequests)
-             .pluginConfig(config.getLegacyConfig());
+             .cfgPluginConfig(config.getLegacyConfig());
```

Comment:
@TolyaTalamanov should probably align these APIs between `ie::Params<>` and `ov::Params<>`